assessment/services/assessment.py

```diff
@@ -110,6 +111,40 @@ def create_response_test(request_data, user):
     return response_test
 
 
+def validate_video_coference_notification(request_data):
+    """
+    Video Conference Notification Must Have Subject, Message, Time Release
+    """
+    if len(request_data.get('subject').split()) == 0:
+        raise InvalidVideoConferenceNotificationException('Subject Should Not Be Empty')
+    if len(request_data.get('message').split()) == 0:
+        raise InvalidVideoConferenceNotificationException('Message Should Not Be Empty')
+
+
+def save_video_conference_notification_to_database(request_data: dict, assessor: Assessor):
+    name = request_data.get('name')
+    description = request_data.get('description')
+    owning_company = assessor.associated_company
+    subject = request_data.get('subject')
+    message = request_data.get('message')
+    assignment = VideoConferenceNotification.objects.create(
+        name=name,
+        description=description,
+        owning_company=owning_company,
+        subject=subject,
+        message=message,
+    )
+    return assignment
+
+
+def create_video_conference_notification(request_data, user):
```

Comment:
Hi Arya if I may, this means that the VideoConferenceNotification will be an assessment tool right? Thus, we will be able to create one to notify the assessee with server sent events?

Reply:
Hi Rashad, yes it is an assessment tool